mbed_lib.json

```diff
@@ -3,7 +3,8 @@
     "config": {
         "storage": {
             "help": "Options are SD_CARD, SPI_FLASH, DATA_FLASH, INTERNAL_FLASH, and HEAP",
-            "required": true
+            "//required": true,
```

Comment:
what is this change?

Reply:
Agh! Good catch

Reply:
Uhhh, I was just testing you guys. Yeah, hahaha. : )